dev/user.clj

```diff
@@ -0,0 +1,82 @@
+(ns dev.user
```

Comment:
Lol, you missed out on a whole conversation re: this yesterday, which is partially why I wanted @otfrom to review. Essentially this is less of a test and more of an analysis of the results produced by the ccm. The idea will be that as we improve the ccm, the values will be closer to the ONS projections. Should have put this in the description, sorry!